packages/components/tool-bar/src/tool-bar.ts

```diff
@@ -0,0 +1,229 @@
+import { faEllipsisVertical } from '@fortawesome/pro-regular-svg-icons';
+import { type ScopedElementsMap, ScopedElementsMixin } from '@open-wc/scoped-elements/lit-element.js';
+import { Button } from '@sl-design-system/button';
+import { ButtonGroup } from '@sl-design-system/button-group';
+import { Icon } from '@sl-design-system/icon';
+import { MenuButton, MenuItem, MenuItemGroup } from '@sl-design-system/menu';
+import { ToggleButton } from '@sl-design-system/toggle-button';
+import { type CSSResultGroup, LitElement, type PropertyValues, type TemplateResult, html, nothing } from 'lit';
+import { property, state } from 'lit/decorators.js';
+import { ToolBarDivider } from './tool-bar-divider.js';
+import styles from './tool-bar.scss.js';
+
+declare global {
+  interface HTMLElementTagNameMap {
+    'sl-tool-bar': ToolBar;
+  }
+}
+
+export interface ToolBarItemBase {
+  element: HTMLElement;
+  visible: boolean;
+}
+
+export interface ToolBarItemButton extends ToolBarItemBase {
+  type: 'button';
+  disabled?: boolean;
+  icon?: string | null;
+  label?: string | null;
+  selectable?: boolean;
+
+  click?(): void;
+}
+
+export interface ToolBarItemDivider extends ToolBarItemBase {
+  type: 'divider';
+}
+
+export interface ToolBarItemGroup extends ToolBarItemBase {
+  type: 'group';
+  buttons: ToolBarItemButton[];
+  label?: string | null;
+  selects?: 'single' | 'multiple';
+}
+
+export type ToolBarItem = ToolBarItemButton | ToolBarItemDivider | ToolBarItemGroup;
+
+Icon.register(faEllipsisVertical);
```

Comment:
didn't we agree that all the icons we use in our own components will be added in the default icon set, so they are always available?
(This is done by adding them in the core.json)

Reply:
I specifically did not add it yet, since it's still a draft version and there is no UI design yet.